shub/image/deploy.py

```diff
@@ -133,10 +133,10 @@ class _LoggedDeployProgress(utils.BaseProgress):
 
     def show(self):
         click.echo("Deploy results:")
-        super(_LoggedDeployProgress, self).show()
+        super().show()
 
     def handle_event(self, event):
-        click.echo("{}".format(event))
+        click.echo(f"{event}")
```

Comment:
If:

```suggestion
        click.echo(event)
```

Does not work, then maybe:

```suggestion
        click.echo(str(event))
```

Reply:
`click.echo` casts args to str so first one works